onward/app/controllers/RichLinkController.scala

```diff
@@ -1,44 +1,32 @@
 package controllers
 
-import play.api.mvc.{Action, Controller, RequestHeader}
-import common.{Edition, ExecutionContexts, JsonComponent, Logging}
-import implicits.Requests
-import model.{ApplicationContext, Cached, Content, ContentType, NoCache}
-import scala.concurrent.Future
-import contentapi.ContentApiClient
 import com.gu.contentapi.client.model.v1.ItemResponse
-import play.twirl.api.HtmlFormat
-
-class RichLinkController(contentApiClient: ContentApiClient)(implicit context: ApplicationContext) extends Controller with Paging with Logging with ExecutionContexts with Requests   {
+import common.{ExecutionContexts, Logging}
+import contentapi.ContentApiClient
+import implicits.Requests
+import model.{ApplicationContext, Content, ContentType}
+import play.api.mvc.{Action, RequestHeader}
 
-  def renderHtml(path: String) = render(path)
+class RichLinkController(contentApiClient: ContentApiClient)(implicit context: ApplicationContext) extends RenderTemplateController(contentApiClient) with Paging with Logging with ExecutionContexts with Requests   {
 
   def render(path: String) = Action.async { implicit request =>
-    lookup(path) map { content =>
-      content.map(renderContent).getOrElse(NotFound)
+    lookup(path) map { response: ItemResponse =>
+      response.content.map(Content(_)) match {
+        case Some(content) => renderContent(richLinkHtml(content), richLinkBodyHtml(content))
+        case None => NotFound
+      }
     }
   }
 
-  private def lookup(path: String)(implicit request: RequestHeader): Future[Option[ContentType]] = {
-    val edition = Edition(request)
-    log.info(s"Fetching article: $path for edition: ${edition.id}:")
-
-    val response: Future[ItemResponse] = contentApiClient.getResponse(
-      contentApiClient.item(path, edition)
-        .showFields("headline,standfirst,shortUrl,webUrl,byline,starRating,trailText,liveBloggingNow")
-        .showTags("all")
-        .showElements("all")
-    )
-
-    response.map { response => response.content.map(Content(_))  }
+  private def lookup(path: String)(implicit request: RequestHeader) = {
+    val fields = "headline,standfirst,shortUrl,webUrl,byline,starRating,trailText,liveBloggingNow"
```

Comment:
Might be a little ugly because of the length of the string but would it be beneficial to have `fields` in the constructor so we can define `lookup` completely in the superclass?

    ... extends RenderTemplateController(contentApiClient,
          fields = "headline,standfirst,shortUrl,webUrl,byline,trailText,liveBloggingNow,commentCloseDate,commentable")